apps/passport-client/new-components/screens/Home/NewHomeScreen.tsx

```diff
@@ -757,3 +817,63 @@ export const NewHomeScreen = (): ReactElement => {
     </AppContainer>
   );
 };
+
```

Comment:
tbh this page becomes kinda big, can move all the zap related stuff to a different page and import it ?